src/components/arrows.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+export function SampleNextArrow(props) {
+  const {
+    className, style, onClick, onKeyPressHandler,
+  } = props;
+
+  return (
+    <div
+      className={className}
+      style={{
```

Comment:
- [ ] It looks lke you're adding style inside the component, why not add in a separate script? 🐒